CHANGELOG.md

```diff
@@ -217,7 +296,7 @@
   [#2083](https://github.com/tigerbeetle/tigerbeetle/pull/2083),
   [#2085](https://github.com/tigerbeetle/tigerbeetle/pull/2085)
 
-  Multiple and varied changes to conform **all** line lengths to not more than 100 columns,
+  ,ple and varied changes to conform **all** line lengths to not more than 100 columns,
```

Comment:
Oops?

Reply:
Ah, that's just [devil](https://susam.github.io/devil/) messing me up!